derive_client/cli.py

```diff
@@ -96,6 +97,56 @@ def cli(ctx, log_level):
     ctx.obj["client"] = set_client(ctx)
 
 
+@cli.group("bridge")
+def bridge():
+    """Interact with bridging functions."""
+
+
+@bridge.command("deposit")
+@click.option(
+    "--chain-id",
+    "-c",
+    type=click.Choice(f"{c.name}" for c in ChainID),
+    required=True,
+    help="The chain ID to bridge FROM.",
+)
+@click.option(
+    "--receiver", "-r", type=str, required=True, help="The Derive smart contract wallet address to receive the funds."
+)
```

Comment:
although in the bridge code it makes sense to have this, integrated with the `BaseClient` we already have our smart wallet address for the funding account set. So, for the cli tool, the `--receiver` could be removed. To clarify
- The bridge client requires `receiver` to be provided; it is general in that sense.
- From the perspective of the derive client, we may assume we want to fund our own wallet and simplify this interface.

What do you think?